cmake/ccf.cmake

```diff
@@ -56,7 +56,7 @@ function(add_san name)
 endfunction()
 
 separate_arguments(COVERAGE_FLAGS UNIX_COMMAND "-fprofile-instr-generate -fcoverage-mapping")
-separate_arguments(COVERAGE_LINK UNIX_COMMAND "-fprofile-instr-generate -fcoverage-mapping")
+separate_arguments(COVERAGE_LINK UNIX_COMMAND "-fprofile-instr-generate -fcoverage-mapping -lgcc")
```

Comment:
I believe that's a dependency for quickjs, but not for coverage, it should be grouped accordingly.